data/src/main/java/com/teamagam/gimelgimel/data/response/entity/GeometryResponse.java

```diff
@@ -0,0 +1,22 @@
+package com.teamagam.gimelgimel.data.response.entity;
+
+import com.teamagam.gimelgimel.data.response.entity.contents.GeoContentData;
+import com.teamagam.gimelgimel.data.response.entity.visitor.IResponseVisitor;
+
+/**
+ * A class for geo messages (using GeoContentData as its content)
```

Comment:
Comments out of sync with the code - make sure you either update them or delete them if they're useless (much like this one).